### PR TITLE
added German translation

### PR DESCRIPTION
**Pull request by lightlike**

### CODE REVIEW

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:aesthetic.12.name=Gewobener Block
+tile.bwm:advanced_bellows.name=Automatischer Blasebalg
+
+tile.bwm:steel_block.name=Block aus seelengeschiedetem Stahl
+tile.bwm:cut_steel_block.name=Zerschnittener Block aus seelengeschiedetem Stahl
+tile.bwm:steel_pressure_plate.name=Stählerne Druckplatte
+
+tile.bwm:windmill.windmill.name=Horizontales Windrad
+tile.bwm:windmill.waterwheel.name=Wasserrad
+tile.bwm:windmill.windmill_vertical.name=Verticales Windrad
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Vertikales Windrad

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:planter.4.name=Blumentopf mit gedüngter Erde
+tile.bwm:planter.5.name=Blumentopf mit Sand
+tile.bwm:planter.6.name=Blumentopf mit Wasser
+tile.bwm:planter.7.name=Blumentopf mit Kies
+tile.bwm:planter.8.name=Blumentopf mit Rotem Sand
+tile.bwm:planter.9.name=Blumentopf mit Erde
+
+tile.bwm:vase.0.name=Weiße Vase
+tile.bwm:vase.1.name=Orangene Vase
+tile.bwm:vase.2.name=Magentane Vase
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Magenta, is Magentane even a word?

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 3]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:vase.10.name=Lilane Vase
+tile.bwm:vase.11.name=Blaue Vase
+tile.bwm:vase.12.name=Braune Vase
+tile.bwm:vase.13.name=Grüne Vase
+tile.bwm:vase.14.name=Rote Vase
+tile.bwm:vase.15.name=Schwarze Vase
+
+tile.bwm:candle.0.name=Weiße Kerze
+tile.bwm:candle.1.name=Orangene Kerze
+tile.bwm:candle.2.name=Magentane Kerze
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Ditto

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+
+tile.bwm:anchor.name=Anker
+tile.bwm:rope.name=Hanfseil
+tile.bwm:hibachi.name=Hibachi
+tile.bwm:hemp.name=Hanfsamen
+tile.bwm:detector.name=Detektor Block
+tile.bwm:lens.name=Fokuslinse
+tile.bwm:vine_trap.name=Rebenfalle
+
+tile.bwm:booster.name=Mechanische Beschleunigungsschine
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: schiene

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 5]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.concentrated_hellfire.name=Konzentriertes Höllenfeuer
+item.bwm:material.coal_dust.name=Kohlestaub
+item.bwm:material.filament.name=Glühfaden
+item.bwm:material.polished_lapis.name=Polierter Lapis Lazuli
+item.bwm:material.potash.name=Pottasche
+item.bwm:material.sawdust.name=Sägemehl
+item.bwm:material.soul_dust.name=Seelenstaub
+item.bwm:material.screw.name=Archimedische Schraube
+item.bwm:material.brimstone.name=Schwefel
+item.bwm:material.niter.name=Sapeter
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Salpeter

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 6]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.element.name=Heizkörper
+item.bwm:material.fuse.name=Explosive Sicherung
+item.bwm:material.blasting_oil.name=Strahlöl
+item.bwm:material.nugget_iron.name=Eisen-Nugget
+item.bwm:material.nugget_steel.name=Nugget aus Seelengeschiedetem Stahl
+item.bwm:material.leather_cut.name=Geschittenes Leder
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Geschnittenes Leder

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 7]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.dung.name=Mist
+item.bwm:material.tanned_leather.name=Gegerbtes Leder
+item.bwm:material.scoured_leather.name=Gereinigtes Leder
+item.bwm:material.leather_strap.name=Ledergurt
+item.bwm:material.leather_belt.name=Lederriemen
+item.bwm:material.wood_blade.name=Holzklinge
+item.bwm:material.windmill_blade.name=Windsegel
+item.bwm:material.glue.name=Kleber
+item.bwm:material.tallow.name=Talg
+item.bwm:material.ingot_steel.name=Barren aus Seelengeschiedetem Stahl
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: geschmiedetem

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 8]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:kibble.name=Trockennahrung
+item.bwm:apple_pie.name=Apfelkuchen
+item.bwm:bat_wing.name=Feldermausflügel
+item.bwm:cooked_bat_wing.name=Gebratener Feldermausflügel
+
+item.bwm:manual.name="Better With Mods" Benutzerhandbuch
+
+item.bwm:axle_generator.windmill.name=Windrad
+item.bwm:axle_generator.waterwheel.name=Wasserrad
+item.bwm:axle_generator.windmill_vertical.name=Verticale Windrad
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Vertikales

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 9]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:hearty_stew.name=Herzhafter Eintopf
+item.bwm:pork_dinner.name=Schweinemahlzeit
+item.bwm:raw_kebab.name=Ungebrahtener Kebab
+item.bwm:wolf_chop.name=Wolfskotlett
+item.bwm:cooked_wolf_chop.name=Gebratenes Wolfskotlett
+item.bwm:mystery_meat.name=Überraschungsfleisch
+item.bwm:cooked_mystery_meat.name=Gebratenes Überraschungsfleisch
+item.bwm:kibble.name=Trockennahrung
+item.bwm:apple_pie.name=Apfelkuchen
+item.bwm:bat_wing.name=Feldermausflügel
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Fledermausflügel

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 10]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:tasty_sandwich.name=Leckeres Sandwich
+item.bwm:beef_dinner.name=Steak-Mahlzeit
+item.bwm:beef_potatoes.name=Steak und Kartoffeln
+item.bwm:chicken_soup.name=Hünersuppe
+item.bwm:chocolate.name=Chokolade
+item.bwm:chowder.name=Eintopf
+item.bwm:cooked_kebab.name=Kebab
+item.bwm:hearty_stew.name=Herzhafter Eintopf
+item.bwm:pork_dinner.name=Schweinemahlzeit
+item.bwm:raw_kebab.name=Ungebrahtener Kebab
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Could be simply "Spieß"

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 11]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:raw_omelet.name=Rohes Pilz-Omlett
+item.bwm:cooked_omelet.name=Gekochtes Pilz-Omlett
+item.bwm:ham_and_eggs.name=Schinken und Eier
+item.bwm:tasty_sandwich.name=Leckeres Sandwich
+item.bwm:beef_dinner.name=Steak-Mahlzeit
+item.bwm:beef_potatoes.name=Steak und Kartoffeln
+item.bwm:chicken_soup.name=Hünersuppe
+item.bwm:chocolate.name=Chokolade
+item.bwm:chowder.name=Eintopf
+item.bwm:cooked_kebab.name=Kebab
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Could be "Gebratener Spieß" or "Grillspieß"

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 12]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:raw_scrambled_egg.name=Rohes Rührei
+item.bwm:cooked_scrambled_egg.name=Gekochtes Rührei
+item.bwm:raw_omelet.name=Rohes Pilz-Omlett
+item.bwm:cooked_omelet.name=Gekochtes Pilz-Omlett
+item.bwm:ham_and_eggs.name=Schinken und Eier
+item.bwm:tasty_sandwich.name=Leckeres Sandwich
+item.bwm:beef_dinner.name=Steak-Mahlzeit
+item.bwm:beef_potatoes.name=Steak und Kartoffeln
+item.bwm:chicken_soup.name=Hünersuppe
+item.bwm:chocolate.name=Chokolade
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Schokolade

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 13]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:cooked_egg.name=Gekochtes Ei
+item.bwm:raw_scrambled_egg.name=Rohes Rührei
+item.bwm:cooked_scrambled_egg.name=Gekochtes Rührei
+item.bwm:raw_omelet.name=Rohes Pilz-Omlett
+item.bwm:cooked_omelet.name=Gekochtes Pilz-Omlett
+item.bwm:ham_and_eggs.name=Schinken und Eier
+item.bwm:tasty_sandwich.name=Leckeres Sandwich
+item.bwm:beef_dinner.name=Steak-Mahlzeit
+item.bwm:beef_potatoes.name=Steak und Kartoffeln
+item.bwm:chicken_soup.name=Hünersuppe
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Hühnersuppe

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 14]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:single_machine.2.name=Gefilterter Trichter
+tile.bwm:single_machine.3.name=Drehscheibe
+tile.bwm:pane.0.name=Gitter
+tile.bwm:pane.1.name=Latten
+tile.bwm:wicker.name=Webgitter
+tile.bwm:grate.0.name=Eichengitter
+tile.bwm:grate.1.name=Fichtengitter
+tile.bwm:grate.2.name=Birkengitter
+tile.bwm:grate.3.name=Tropengitter
+tile.bwm:grate.4.name=Akatiengitter
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Akazien

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 15]
```diff
@@ -0,0 +1,564 @@
...
+
+subtitles.bwm.wood.creak=Holz knarrt
+subtitles.bwm.stone.grind=Steine mahlen
+subtitles.bwm.wood.bellow=Blasebalg bläst
+subtitles.bwm.wood.chime=Hölzernes Windspiel
+subtitles.bwm.metal.chime=Metallenes Windspiel
+subtitles.bwm.oof=Uff
+death.attack.growth=%s wurde vom Wachstum verschlungen.
+death.attack.gloom=%s wurde vom Grue gegessen.
+death.attack.saw=%s hat eine Finger verloren.
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: einen Finger

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 16]
```diff
@@ -0,0 +1,564 @@
...
+
+item.bwm:steel_helmet.name=Plattenhelm
+item.bwm:steel_chest.name=Plattenpanzer
+item.bwm:steel_pants.name=Plattenschienen
+item.bwm:steel_boots.name=Plattenstiefel
+
+item.bwm:leather_tanned_helmet.name=Gegerbte Kappe
+item.bwm:leather_tanned_chest.name=Gegerbter Waffenrock
+item.bwm:leather_tanned_pants.name=Gegerbte Hose
+item.bwm:leather_tanned_boots.name=Gegerbte Sitefel
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Stiefel

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 17]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.ender_slag.name=Ender-Slacke
+item.bwm:material.soul_flux.name=Seelenfluss
+item.bwm:material.ender_ocular.name=Okular des Enders
+item.bwm:material.padding.name=Polsterung
+item.bwm:material.armor_plate.name=Plattenrüstung
+item.bwm:material.broadhead.name=Breitspitze
+item.bwm:material.cocoa_powder.name=Kakaopulver
+item.bwm:material.diamond_ingot.name=Diamantbarren
+item.bwm:material.diamond_nugget.name=Diamantnugget
+item.bwm:material.chain_mail.name=Kette
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Kettengeflecht

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 18]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.scoured_leather_cut.name=Geschittenes Leder (Gereinigt)
+item.bwm:material.redstone_latch.name=Redstone Verriegelung
+item.bwm:material.nether_sludge.name=Nether-Slacke
+item.bwm:material.flour.name=Mehl
+item.bwm:material.haft.name=Heft
+item.bwm:material.soul_dust.name=Seelenstaub
+item.bwm:material.charcoal_dust.name=Holzkohlestaub
+item.bwm:material.sharpening_stone.name=Wetzstein
+item.bwm:material.knife_blade.name=Messerklinge
+item.bwm:material.ender_slag.name=Ender-Slacke
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Schlacke

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 19]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.filament.name=Glühfaden
+item.bwm:material.polished_lapis.name=Polierter Lapis Lazuli
+item.bwm:material.potash.name=Pottasche
+item.bwm:material.sawdust.name=Sägemehl
+item.bwm:material.soul_dust.name=Seelenstaub
+item.bwm:material.screw.name=Archimedische Schraube
+item.bwm:material.brimstone.name=Schwefel
+item.bwm:material.niter.name=Sapeter
+item.bwm:material.element.name=Heizkörper
+item.bwm:material.fuse.name=Explosive Sicherung
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Lunte or Zündschnur

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 20]
```diff
@@ -0,0 +1,564 @@
...
+item.bwm:material.nethercoal.name=Netherkohle
+item.bwm:material.hemp.name=Hanf
+item.bwm:material.hemp_fibers.name=Hanffasern
+item.bwm:material.hemp_cloth.name=Hanftuch
+item.bwm:material.dung.name=Mist
+item.bwm:material.tanned_leather.name=Gegerbtes Leder
+item.bwm:material.scoured_leather.name=Gereinigtes Leder
+item.bwm:material.leather_strap.name=Ledergurt
+item.bwm:material.leather_belt.name=Lederriemen
+item.bwm:material.wood_blade.name=Holzklinge
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Not sure about this one. Possibly Holzschaufel, Wasserrad Schaufel or Holzspeiche?

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 21]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:blood_leaves.name=Blutblätter
+tile.bwm:blood_sapling.name=Blutsetzling
+
+tile.bwm:nether_clay.name=Nether-Schlacke-Block
+
+tile.bwm:cobble.0.name=Granit-Pflasterstein
+tile.bwm:cobble.1.name=Diorit-Pflasterstein
+tile.bwm:cobble.2.name=Andesit-Pflasterstein
+
+tile.bwm:deadweight.name=Tragfähigkeit
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Leergewicht

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 22]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:deadweight.intact.name=Tragfähigkeit
+tile.bwm:deadweight.slightDamaged.name=Leicht Beeinträchtigte Tragfähigkeit
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Leicht beschädigtes Leergewicht

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang [hunk 23]
```diff
@@ -0,0 +1,564 @@
...
+tile.bwm:deadweight.veryDamaged.name=Stark Beeinträchtigte Tragfähigkeit
```
Comment (DaedalusGame) [inline src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Schwer beschädigtes Leergewicht

File: src/main/resources/assets/betterwithmods/lang/de_DE.lang (hunk 21 above)
Comment (lightlike) [reply src/main/resources/assets/betterwithmods/lang/de_DE.lang]: I just looked into that and I think none of those translations really fit in German. I'll try to think of a better one.

Comment (lightlike) [reply src/main/resources/assets/betterwithmods/lang/de_DE.lang]: Changed to "Klotz am Bein" I think that works better.

**Pull request merged**